### DISCUSSION TITLE
Crop to match

### DISCUSSION THREAD

**Discussion by kenrickschulze** [Project-MONAI/MONAI#7558] [Q&A] [answered]
Hey Community,

is there a MONAI transform like ResampleToMatchd but for cropping?

Lets say i have two volumes and one is just a small patch within the bigger volume. As both volumes have center and spacing coordinates, is there a monai transform to overlap both images in pixelspace by cropping the bigger volume to the field of view of the smaller one.

Best wishes :)

**Comment by KumoLiu** [comment, marked as answer]
Hi @kenrickschulze, thanks for your interest here.

At present, we don't offer support for the "crop to match" operation. However, you could use the `SpatialCrop` which admits an "roi_center" parameter. By using "roi_center", you can specify the center for cropping your data.
https://github.com/Project-MONAI/MONAI/blob/ec63e068fec85dff5f043d73f2c029b359332707/monai/transforms/croppad/array.py#L462

Hope it helps, thanks.

**Reply by kenrickschulze** [reply]
Hey @KumoLiu 

thank you for ur reply. I think it would be  a cool feature for multi modal or registration tasks :)
